src/test/java/eu/bitwalker/useragentutils/browser/XboxParameterizedTest.java

```diff
@@ -0,0 +1,33 @@
+package eu.bitwalker.useragentutils.browser;
+
+import static java.util.Arrays.asList;
+
+import java.util.Collection;
+
+import org.junit.runners.Parameterized.Parameters;
+
+import eu.bitwalker.useragentutils.Browser;
+
+public class XboxParameterizedTest extends AbstractUserAgentParameterizedTest {
+
+	public XboxParameterizedTest(String userAgentValue,
+			Browser expectedBrowser, String browserVersionExpected,
+			String osExpected) {
+		super(userAgentValue, expectedBrowser, browserVersionExpected,
+				osExpected);
+	}
+
+	@Parameters
+	public static Collection<Object[]> testData() {
+		return asList(new Object[][] {
+
+				{
+						"Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; Trident/6.0; Xbox; Xbox One)",
```

Comment:
Pull the latest changes from trunk. I've refactored the AbstractUserAgentParameterizedTest to use the OS enum instead of a string value.